dll/win32/comctl32/listview.c

```diff
@@ -7502,6 +7505,7 @@ static INT LISTVIEW_GetNextItem(const LISTVIEW_INFO *infoPtr, INT nItem, UINT uF
           }
           return -1;
         }
+#endif
```

Comment:
Should this be disabled for all listviews, or just the one used for the `CDefView`?
Maybe @yagoulas has an idea about this?

Reply:
I think it should be for all listviews. It affects every application that uses listview auto-arranging.